docker/Dockerfile

```diff
@@ -2,30 +2,18 @@ ARG UBUNTU_VERSION=18.04
 FROM ubuntu:${UBUNTU_VERSION}
 
 # Use bash as the default shell
-RUN ln -snf /bin/bash /bin/sh
+SHELL ["/bin/bash", "-c"]
 
 # Update core packages
 RUN apt-get -y update
-RUN apt-get -y upgrade
-RUN apt-get -y dist-upgrade
 
 # Get some apt packages
 RUN apt-get -y install autoconf clang cmake g++ gcc gfortran git libblas-dev \
                        libhdf5-dev liblapack-dev libpython2.7-dev libtool \
                        python-numpy python-pip python-setuptools wget
 
-# Get astyle
```

Comment:
Why was astyle moved out of this file? Everything that could possibly mess with system libraries (i.e. apt or pip) was put in this file.

Reply:
I put it back in :)